pkg/registries/registries_test.go

```diff
@@ -244,11 +243,10 @@ func TestMergedICSPMirrorSets(t *testing.T) {
 					},
 				})
 			}
-			res, err := mergedICSPMirrorSets(in)
+			res, err := mergedDigestMirrorSets(nil, in)
```

Comment:
Very non-blocking: If the goal is to exhaustively unit-test `merged{ICSP,Digest}MirrorSets`, it might be useful to test also cases where both inputs are non-empty.

Maybe merge the two unit test functions, and run them in three modes:
- all `mergedMirrorsetsTestcases` are ICSP
- all `mergedMirrorsetsTestcases` are IDMS
- odd `mergedMirrorsetsTestcases` are ICSP, even are IDMS (IIRC this transformation does not change semantics)

Reply:
Fixed.